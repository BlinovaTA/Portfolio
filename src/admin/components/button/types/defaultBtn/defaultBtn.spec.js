import defaultBtn from "./defaultBtn.vue";
import { mount } from "@vue/test-utils";

it("default button snapshot", async () => {
  const mountWrapper = mount(defaultBtn);
  expect(mountWrapper.element).toMatchSnapshot();
});