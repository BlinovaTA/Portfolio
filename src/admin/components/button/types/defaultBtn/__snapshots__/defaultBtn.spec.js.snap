// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default button snapshot 1`] = `
<button
  class="default-btn-container btn-decorator"
>
  Отправить
</button>
`;
